_posts/2017-09-25-saveourhealthcare-home.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: petition-page
+permalink: /saveourhealthcare/
+title: "Tell Congress: Stop playing with our Lives"
+category: campaigns
+featured-image: '/static/images/featured/healthcare-kimmel.png'
```

Comment:
Image paths are case-sensitive because of AWS. Please fix this to match the case in the filename.